core/MySigningAtsha204.cpp

```diff
@@ -35,26 +36,25 @@
 #define SIGN_DEBUG(x,...)
 #endif
 
-// Define MY_DEBUG_VERBOSE_SIGNING in your sketch to enable signing backend debugprints
-
-unsigned long _signing_timestamp;
-bool _signing_verification_ongoing = false;
-uint8_t _signing_verifying_nonce[NONCE_NUMIN_SIZE_PASSTHROUGH+SHA204_SERIAL_SZ+1];
-uint8_t _signing_signing_nonce[NONCE_NUMIN_SIZE_PASSTHROUGH+SHA204_SERIAL_SZ+1];
-uint8_t _signing_temp_message[SHA_MSG_SIZE];
-uint8_t _signing_rx_buffer[SHA204_RSP_SIZE_MAX];
-uint8_t _signing_tx_buffer[SHA204_CMD_SIZE_MAX];
-extern uint8_t _doWhitelist[32];
-
+static unsigned long _signing_timestamp;
+static bool _signing_verification_ongoing = false;
+static uint8_t _signing_verifying_nonce[32+9+1];
```

Comment:
And the reason for removing the defines, is that I try to align both backends to be very similar so it is easy to diff them.